polyfill/test/time.mjs

```diff
@@ -249,14 +250,19 @@ describe('Time', () => {
         end = start2.plus(duration);
         equal(`${start2.difference(end)}`, 'PT11H');
       });
+      */
       it('returns the same duration no matter when the start time is', () => {
         const hours = Array(24)
           .fill()
           .map((_, ix) => ix);
         hours.forEach((hour) => {
           const time1 = new Temporal.Time(hour);
           const time2 = time1.plus(Temporal.Duration.from('PT9H'));
-          equal(`${time1.difference(time2)}`, 'PT9H');
```

Comment:
These tests should probably be rewritten since the description ("returns the same duration no matter when the start time is") no longer applies.

Reply:
I already partially rewrote these tests (in the original commit) to run the original test logic if the values were in the correct order, and to validate throwing if the values were reversed-- so I think I'm covering both the original intent and the new behavior. Happy to revise, but not sure what's missing. What else do you think would be helpful to have in these tests?

Reply:
What I meant was that we should not have this big loop where half the values are out of order and will throw. The test was written specifically to test the looping-around behaviour, so if we no longer do that, then the test can just be removed. Instead we should add a separate test to verify the throwing behaviour.

Reply:
OK sounds good. I removed those looping tests and replaced with a simple test to verify out-of-order throwing.  Resolving.